cluster-autoscaler/cloudprovider/azure/azure_cache.go

```diff
@@ -368,6 +370,16 @@ func (m *azureCache) FindForInstance(instance *azureRef, vmType string) (cloudpr
 	return nil, nil
 }
 
+// isAllScaleSetsAreUniform determines if all the scale set autoscaler is monitoring are Uniform or not.
+func (m *azureCache) areAllScaleSetsAreUniform() bool {
```

Comment:
```suggestion
func (m *azureCache) areAllScaleSetsUniform() bool {
```
(same suggestion as before, may have been misread)